source/brailleDisplayDrivers/brailliantB.py

```diff
@@ -182,11 +195,11 @@ def _serHandleResponse(self, msgId, payload):
 		else:
 			log.debugWarning("Unknown message: id {id!r}, payload {payload!r}".format(id=msgId, payload=payload))
 
-	def _hidOnReceive(self, data):
+	def _hidOnReceive(self, data: bytes):
 		rId = data[0]
 		if rId == HR_KEYS:

```

Comment:
Woops, another fail of the linter I presume?

Reply:
Yeah, it doesn't pick up these cases at all, its legal to allow comparison of int and bytes, but it will always be false. Given that you'd think there would be a warning though! I don't think I realised that the linter missed it until one of the later drivers, so I probably wasn't as careful here,  though it is easy to miss. Drivers are done one per commit, so hopefully I caught these in the later ones. I'm still only about half way through!